npc_simulator/src/node.cpp

```diff
@@ -90,7 +90,7 @@ NPCSimulator::NPCSimulator(rclcpp::Node & node)
     "output/debug_object_info", durable_qos);
 
   // register callback
-  engage_sub_ = node.create_subscription<std_msgs::msg::Bool>(
+  engage_sub_ = node.create_subscription<autoware_vehicle_msgs::msg::Engage>(
     "input/engage", large_queue_size, std::bind(&NPCSimulator::engageCallback, this, _1));
   object_sub_ = node.create_subscription<npc_simulator::msg::Object>(
     "/simulation/npc_simulator/object_info", large_queue_size, std::bind(&NPCSimulator::objectCallback, this, _1));
```

Comment:
This Line is longer than 100 characters. Could you apply reformat by ament_uncrustify?